test/user.js

```diff
@@ -0,0 +1,139 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  logger = require('../app/logger'),
+  should = chai.should();
+
+const chaiPost = (path, object) =>
+  chai
+    .request(server)
+    .post(path)
+    .send(object);
+
+describe('/users POST', () => {
+  it('should fail because first name is missing', done => {
+    chai
+      .request(server)
+      .post('/users')
+      .send({
+        lastName: 'lastName',
+        password: 'password',
+        email: 'email@wolox.com.ar'
+      })
+      .catch(err => {
+        err.should.have.status(400);
+        err.response.should.be.json;
+        err.response.body.should.have.property('message');
+        err.response.body.should.have.property('internal_code');
+      })
+      .then(() => done());
```

Comment:
Change this in all tests, with this `.then` you can avoid `.catch` and never fail